api/libretime_api/storage/tests/views/test_file.py

```diff
@@ -61,3 +61,32 @@ def test_destroy_invalid(self):
         file_id = "1"
         response = self.client.delete(f"/api/v2/files/{file_id}")
         self.assertEqual(response.status_code, 404)
+
+    def test_filters_limits(self):
+        file = baker.make(
+            "storage.File",
+            mime="audio/mp3",
+            filepath=AUDIO_FILENAME,
+            genre="Soul",
+            md5="5a11ffe0e6c6d70fcdbad1b734be6482",
+        )
+        baker.make(
+            "storage.File",
+            mime="audio/mp3",
+            filepath=AUDIO_FILENAME,
+            genre="R&B",
+            md5="5a11ffe0e6c6d70fcdbad1b734be6483",
+        )
+        self.client.credentials(HTTP_AUTHORIZATION=f"Api-Key {self.token}")
+
+        path = "/api/v2/files"
+        results = self.client.get(path).json()
+        self.assertEqual(len(results), 2)
+
+        path = f"/api/v2/files?md5={file.md5}"
+        results = self.client.get(path).json()
+        self.assertEqual(len(results), 1)
+
+        path = "/api/v2/files?genre=Soul"
```

Comment:
Does the old failure here for R&B imply that you cannot filter on genres with & in them?

Reply:
Not really. I wrote "R&B" but one would have to properly encode the & to %26. I guess browsers will by default convert the space to %20. So if we used this in the front end or via the API, you'd have to make sure the query params are url encoded properly. I guess I was being lazy and thought to just change to "Soul"